### PR TITLE
🏷️ Export props interface

### PR DESCRIPTION
**Pull request by johannchopin**
Hey @SeanMcP 👋 Thanks for your work on this package.

I improved it by enabling the import of the component props. Usefull to reuse it in the TS project.

### CODE REVIEW

**Comment by SeanMcP** [comment]
This is a good idea! I want the add a test case to ensure that the type is available for import (mostly to make sure that I don't accidentally remove it later). ~~Do you have any suggestions of how to do that?~~ See review suggestion below.

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1 @@
-import Emoji from './Emoji';
-
-export default Emoji;
+export * from './Emoji';
```
Comment (SeanMcP) [inline src/index.ts:1]: TypeScript tells me that: 'export *' does not re-export a default.

I would prefer not to change the exporting API, so can we change this to export the component as default and everything else as named?

**Review by SeanMcP — CHANGES_REQUESTED** [review]
Some other things that I would like to see in this PR:
- [x] Change the import for the test file from `./Emoji` to `./index`. This will run tests against the library's entry point as opposed to the component file.
- [x] Import the new type into the test file and apply it to the `props` object. This will use TypeScript to ensure that the export is included and that the test is passing props appropriately.
- [x] A line in the CHANGELOG noting that types interface is now exported. This will go into a minor release.

File: src/index.ts (hunk 1 above)
Comment (johannchopin) [reply src/index.ts:1]: @SeanMcP Oups I was a bit fast for this PR sorry for that 😅 For sure we should keep the current exporting API. I also implemented the other suggestions 👍

**Review by SeanMcP — APPROVED** [review]
Looks good to me! Feel free to tag yourself in the CHANGELOG if you'd like the credit.

I'll merge these changes and have a new release ready in the coming days.

**Comment by johannchopin** [comment]
@SeanMcP Ready to be merged sir 😉

**Comment by SeanMcP** [comment]
Thanks for the contribution, @johannchopin! 🎉

**Comment by SeanMcP** [comment]
⚡ The change is live: https://www.npmjs.com/package/a11y-react-emoji/v/1.2.0

**Pull request merged**